apps/estuary/assets/static/robots.txt

```diff
@@ -0,0 +1,5 @@
+# See http://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
```

Comment:
What is this file for?

Reply:
We don't, but that's a story for another day.